app/controllers/application_controller.rb

```diff
@@ -105,6 +105,10 @@ def instance_id
     params[:instance_id]
   end
 
+  def organization_name
+    params.dig(:context, :organization_name)
+  end
+  
```

Comment:
Just a nit, Looks like an extra space, only if you have to make other changes.

Reply:
fixed